hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -523,7 +523,8 @@ public boolean syncDataFromOM() {
             Thread.currentThread().interrupt();
           } catch (Exception e) {
             metrics.incrNumDeltaRequestsFailed();
-            LOG.warn("Unable to get and apply delta updates from OM.", e);
+            LOG.warn("Unable to get and apply delta updates from OM.",
```

Comment:
> I have a question! Would it be a good idea to mention the `this.getClass().getName()` to over here, since the stack trace would not be printed?

Whenever there is a log printed in logs, log4j always prints the class name from where this log has appeared, so I believe that is not needed.